facebook-commerce.php

```diff
@@ -2994,7 +2969,7 @@ public function is_messenger_enabled() {
 		 * @param bool $is_enabled whether the Facebook messenger is enabled
 		 * @param \WC_Facebookcommerce_Integration $integration the integration instance
 		 */
-		return (bool) apply_filters( 'wc_facebook_is_messenger_enabled', 'yes' === get_option( self::SETTING_ENABLE_MESSENGER ), $this );
+		return (bool) apply_filters( 'wc_facebook_is_messenger_enabled', 'yes' === get_option( self::SETTING_ENABLE_MESSENGER, 'yes' ), $this );
```

Comment:
Shouldn't Messenger be disabled by default?

Reply:
whoops!